packages/web-components/src/components/cbp-form-field-wrapper/cbp-form-field-wrapper.stories.tsx

```diff
@@ -74,8 +76,119 @@ InputWithOverlays.args = {
+
+    //console.log({e});
+
+    let value:number = Number(input.value) || 0;
+    if(button.getAttribute('name')==="increment") {
+      input.value = `${value + step}`;
+    }
+    if(button.getAttribute('name')==="decrement") {
+      input.value = `${value - step}`;
+    }
+  });
+  
+
+  return ` 
+    <cbp-form-field
+      ${label ? `label="${label}"` : ''}
+      ${description ? `description="${description}"` : ''}
+      ${fieldId ? `field-id="${fieldId}"` : ''}
+      ${error ? `error` : ''}
+      ${readonly ? `readonly` : ''}
+      ${disabled ? `disabled` : ''}
+      ${context && context != 'light-inverts' ? `context=${context}` : ''}
+      ${sx ? `sx=${JSON.stringify(sx)}` : ''}
+    >
+      <cbp-form-field-wrapper
+        ${sx ? `sx=${JSON.stringify(sx)}` : ''}
+      >
+        <input
+          type="${inputType}"
+          name="search"
+          ${value ? `value="${value}"` : ''}
+        />
+
+        ${overlayStart != undefined ? `<span slot="cbp-form-field-overlay-start">${overlayStart}</span>` : ''}
+        ${overlayEnd != undefined ? `<span slot="cbp-form-field-overlay-end">${overlayEnd}</span>` : ''}
+        
+        <span slot="cbp-form-field-unattached-buttons">
+          <cbp-button
+            name="decrement"
+            type="button"
+            fill="outline"
+            color="secondary"
+            variant="square"
+            accessibility-text="Toggle visibility"
+            aria-controls="${fieldId}"
+          >
+            <cbp-icon name="minus" size="1rem"></cbp-icon>
+          </cbp-button>
+
+          <cbp-button
+            name="increment"
+            type="button"
+            fill="outline"
+            color="secondary"
+            variant="square"
+            accessibility-text="Toggle visibility"
+            aria-controls="${fieldId}"
+          >
+            <cbp-icon name="plus" size="1rem"></cbp-icon>
+          </cbp-button>
+
+        </span>
+        
+      </cbp-form-field-wrapper>
+    </cbp-form-field>
+  `;
+};
+
+export const NumericCounter = NumericCounterTemplate.bind({});
+NumericCounter.args = {
+  label: 'Numeric Counter Field',
+  description: '',
+  fieldId: 'numeric-input',
+  inputType: 'number',
+  value: '',
+};
+
+
+
 const PasswordTemplate = ({ label, description, inputType,  overlayStart, overlayEnd, fieldId, error, readonly, disabled, value, context, sx }) => {
+
+  // Ideally, this should be placed on the button component itself, not the document; but the event bubbles, so it works here.
+  document.addEventListener('buttonClick', function(e) {
+    const buttonComponent = e.target as HTMLCbpButtonElement;
+    const button = buttonComponent.querySelector('button') as HTMLButtonElement;
+    const buttonIcon = buttonComponent.querySelector('cbp-icon');
+    const input = document.querySelector(`#${fieldId}`) || document.querySelector('input');
+    const type = input.getAttribute('type');
+
+    //console.log({e});
```

Comment:
I assume this would be nice for testing later but wanted to make sure you meant to leave it

Reply:
removed